pilot/pkg/networking/networking_test.go

```diff
@@ -117,3 +133,166 @@ func TestModelProtocolToListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func TestMakeTunnelAbility(t *testing.T) {
+	tests := []struct {
+		name  string
+		value TunnelType
+		want  TunnelAbility
+	}{
+		{
+			"tunnelAbility H2Tunnel",
+			H2Tunnel,
+			1,
+		},
+		{
+			"tunnelAbility NoTunnel",
+			NoTunnel,
+			0,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := MakeTunnelAbility(tt.value); got != tt.want {
+				t.Errorf("Failed to get MakeTunnelAbility: got = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestMessageToAny(t *testing.T) {
+	tests := []struct {
+		name  string
+		value anypb.Any
+		want  *anypb.Any
+	}{
+		{
+			"messageToAny TcpProxy",
+			anypb.Any{},
+			&anypb.Any{},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := MessageToAny(&tt.value); got == tt.want {
+				t.Errorf("Failed to get MessageToAny: got = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestToString(t *testing.T) {
+	tests := []struct {
+		name  string
+		value TunnelType
+		want  string
+	}{
+		{
+			"H2Tunnel",
+			H2Tunnel,
+			H2Tunnel.ToString(),
+		},
+		{
+			"NoTunnel",
+			NoTunnel,
+			NoTunnel.ToString(),
+		},
```

Comment:
same

Reply:
sure @hzxuzhonghu , i have updated test cases. please review and approve